front/src/routes/dashboard/style.css

```diff
@@ -25,18 +25,18 @@
 
 @media (min-width: 992px) {
   .removePaddingFirstCol {
-    padding-left: 0;
+    padding-left: 0 !important;
   }
 
   .removePaddingLastCol {
-    padding-right: 0;
+    padding-right: 0 !important;
   }
 }
 
 @media (max-width: 992px) {
   .removePadding {
-    padding-left: 0;
-    padding-right: 0;
+    padding-left: 0 !important;
+    padding-right: 0 !important;
```

Comment:
Is important everywhere really needed? Can't you do without?

Reply:
There are just 3 cases here : 

1. 1 column
2. 2 column
3. 3 column

Maybe it could just be 3 CSS classes to avoid avoid inline styles generated by Javascript?